operations/helm/charts/mimir-distributed/values.yaml

```
@@ -1937,6 +1937,9 @@ metaMonitoring:
           labelSelectors:
             app.kubernetes.io/name: kube-state-metrics
 
+      # -- The scrape interval for all ServiceMonitors.
+      scrapeInterval: 30s
```

60s is the default in grafana cloud free, that's why I left it as 60s by default. I will keep it like that 
```suggestion
      scrapeInterval: 60s
```